### PR TITLE
feat(prysm): arm support

### PR DESCRIPTION
**Pull request by dzmitry-lahoda**
for https://github.com/nix-community/ethereum.nix/issues/284

Seems libs i patched are old, devs improved arm build in master. may be should update.

```
actions-runner@Ubuntu-2204-jammy-arm64-base:~/github.com/nix-community/ethereum.nix$ nix build .#prysm -L --show-trace
actions-runner@Ubuntu-2204-jammy-arm64-base:~/github.com/nix-community/ethereum.nix$ uname -a
Linux Ubuntu-2204-jammy-arm64-base 5.15.0-83-generic #92-Ubuntu SMP Mon Aug 14 09:34:05 UTC 2023 aarch64 aarch64 aarch64 GNU/Linux
actions-runner@Ubuntu-2204-jammy-arm64-base:~/github.com/nix-community/ethereum.nix$ 
```

### CODE REVIEW

File: packages/libs/mcl/default.nix [hunk 1]
```diff
@@ -2,6 +2,10 @@
   fetchFromGitHub,
   gmpxx,
   stdenv,
+  clang,
+  cmake,
+  system,
+  pkgs,
```
Comment (aldoborrero) [inline packages/libs/mcl/default.nix:8]: @dzmitry-lahoda Incorporate lib directly rather than using pkgs

File: packages/libs/bls/default.nix [hunk 2]
```diff
@@ -3,6 +3,9 @@
   fetchFromGitHub,
   gmp,
   stdenv,
+  clang,
+  system,
+  pkgs,
```
Comment (aldoborrero) [inline packages/libs/bls/default.nix:8]: @dzmitry-lahoda use lib instead of importing pkgs

**Review by aldoborrero — CHANGES_REQUESTED** [review]
Minor changes are requested that are mostly related to importing directly `lib` rather than `pkgs.lib`.

**Comment by aldoborrero** [comment]
@dzmitry-lahoda thanks for contributing! Apart from the minor changes I requested, would you mind squashing all your commits and updating the commit message to be more like: "feat: add arm support to prysm"?

**Comment by dzmitry-lahoda** [comment]
oh, sorry. missed. was occupied by ethereum testnet release. may be you used squash merge setting in github?

**Pull request merged**